lightningd/notification.c

```diff
@@ -131,3 +149,19 @@ void notify_forward_event(struct lightningd *ld,
 	jsonrpc_notification_end(n);
 	plugins_notify(ld->plugins, take(n));
 }
+
+void notification_call(struct lightningd *ld, const char* topic,
+		       void *payload)
+{
+	struct notification *noti = find_notification_by_topic(topic);
+	assert(noti);
```

Comment:
Nit: It's likely better to changes this into `if (noti == NULL) { fatal("Could not find notification topic %s", topic); }` just to make it more obvious later.

Reply:
Thank you, it's better!

Reply:
Fixed.